### PR TITLE
Use real data for getLedgerConfiguration

### PR DESCRIPTION
**Pull request by rautenrieth-da**
`getLedgerConfiguration()` used to return only the initial ledger config, as specified in the ledger API server config.

This PR changes the call to return real data, as specified by the ledger or `ReadService`.

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/index/LedgerBackedIndexService.scala [hunk 1]
```diff
@@ -270,6 +270,15 @@ abstract class LedgerBackedIndexService(
           .getOrElse(Future.successful(None)))
       .flatMapConcat(ledger.configurationEntries(_).map(_._2.toDomain))
 
+  override def getLedgerConfiguration(): Source[LedgerConfiguration, NotUsed] =
```
Comment (SamirTalwar-DA) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/index/LedgerBackedIndexService.scala]: May be out of scope, but should this also take a `startExclusive` parameter?

Comment (gerolf-da) [reply ledger/sandbox/src/main/scala/com/digitalasset/platform/index/LedgerBackedIndexService.scala]: No, but the implementation should emit the latest configuration and then publish updates. But honestly, configurations change very seldom, so this shouldn't really be a streaming response.

Comment (SamirTalwar-DA) [reply ledger/sandbox/src/main/scala/com/digitalasset/platform/index/LedgerBackedIndexService.scala]: I agree with that last sentence.

**Pull request closed without merging**